lib/reline/line_editor.rb

```diff
@@ -1571,11 +1571,13 @@ def wrap_method_call(method_symbol, method_obj, key, with_operator = false)
       return if key.char >= 128 # maybe, first byte of multi byte
       method_symbol = @config.editing_mode.get_method(key.combined_char)
       if key.with_meta and method_symbol == :ed_unassigned
-        # split ESC + key
-        method_symbol = @config.editing_mode.get_method("\e".ord)
-        process_key("\e".ord, method_symbol)
-        method_symbol = @config.editing_mode.get_method(key.char)
-        process_key(key.char, method_symbol)
+        if @config.editing_mode_is?(:vi_command) || @config.editing_mode_is?(:vi_insert)
+          # split ESC + key in vi mode
```

Comment:
This is only used in vi mode.
In emacs mode, this is making ESC+char inserted into input buffer